content/millennium/r4/scheduling/appointment.md

```diff
@@ -160,7 +173,7 @@ _Implementation Notes_
 
 <%= patch_definition_table(:appointment_patch, :r4) %>
 
-### Example
+### Patch Status Example
```

Comment:
as above let's leave this as Example

Reply:
https://github.com/cerner/fhir.cerner.com/pull/294/commits/2662d3989c06b836a807167951a85a63068e21b5#diff-1d7df8ceb9f0ae8471323516e037c662R175